test/functional/p2p_opportunistic_1p1c.py

```diff
@@ -374,6 +389,174 @@ def test_other_parent_in_mempool(self):
         result_missing_parent = node.submitpackage(package_hex_missing_parent)
         assert_equal(result_missing_parent["package_msg"], "package-not-child-with-unconfirmed-parents")
 
+    def create_large_orphan(self):
+        """Create huge orphan transaction"""
+        tx = CTransaction()
+        # Nonexistent UTXO
+        tx.vin = [CTxIn(COutPoint(random.randrange(1 << 256), random.randrange(1, 100)))]
+        tx.wit.vtxinwit = [CTxInWitness()]
+        tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_NOP] * 390000)]
```

Comment:
in commit d4b787b25f07b212bedb26433e64378673a27f6a:
```suggestion
        tx.wit.vtxinwit[0].scriptWitness.stack = [b'X' * 390000]
```
for a significant orphan-tx creation speed-up (~30s vs ~30ms on my machine for the `large_orphans` list creation at the call-site)

could also move the orphan creation helpers to `mempool_util.py`, since the large one is defined in `p2p_orphan_handling.py` as well